index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./index.css" rel="stylesheet" type="text/css">
+    <title>Awesome books</title>
+</head>
+<body>
+    <header>
+        <section class="heading">
+            <h1>Awesome Books</h1>
+        </section>
+        <nav>
+            <ul class="nav-bar">
+                <li class="item1"><a href="#">List</a></li>
+                <li class="item2"><a href="#">Add New</a></li>
+                <li class="item3"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <div id="current-time-zone">
+            <div id="date-time"></div>
+          </div>          
```

Comment:
- [ ] Please align your `current-time-zone` container properly because right now it is not looking good and professional, your container appears on the top of the navigation menu. Kindly fix the spacing issue and put the container in the right place or position.

### Requirement (Look at the top level text in the below image)
![Screenshot from 2023-08-22 14-18-55](https://github.com/DevKaranJ/AwesomeBooks-ES6/assets/66476329/6a0de1d5-6a59-4f3c-b99d-e54cd24bca28)


### Your Design

![Screenshot from 2023-08-22 14-19-41](https://github.com/DevKaranJ/AwesomeBooks-ES6/assets/66476329/01a09ffa-7084-4e02-9f12-5dfe221c0d7c)